aleph2_data_model/src/com/ikanow/aleph2/data_model/interfaces/data_access/IAccessContext.java

```diff
@@ -30,9 +30,43 @@
 /** A context library that is always passed to the AccessModules via injection 
  * @author acp
  */
-public interface IAccessContext {
 
-	//TODO (ALEPH-1):
+import com.google.inject.Injector;
+import com.ikanow.aleph2.data_model.interfaces.data_services.IColumnarService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.IDocumentService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.IGeospatialService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.IGraphService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.IManagementDbService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.ISearchIndexService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.IStorageService;
+import com.ikanow.aleph2.data_model.interfaces.data_services.ITemporalService;
+import com.ikanow.aleph2.data_model.interfaces.shared_services.ISecurityService;
+
+/**
+ * Gives an app access to all the currently configured data services.
+ * 
+ * @author Burch
+ *
+ */
+public interface IAccessContext {	
```

Comment:
lol nvm I'm an idiot then

Alex Piggott
Director of Product Engineering | IKANOW
M: 443.310.5053
E:  apiggott@ikanow.com
W: www.ikanow.com


On Thu, May 7, 2015 at 11:24 AM, cburch <notifications@github.com> wrote:

> In
> aleph2_data_model/src/com/ikanow/aleph2/data_model/interfaces/data_access/IAccessContext.java
> <https://github.com/IKANOW/Aleph2/pull/1#discussion_r29862783>:
>
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.IDocumentService;
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.IGeospatialService;
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.IGraphService;
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.IManagementDbService;
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.ISearchIndexService;
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.IStorageService;
> > +import com.ikanow.aleph2.data_model.interfaces.data_services.ITemporalService;
> > +import com.ikanow.aleph2.data_model.interfaces.shared_services.ISecurityService;
> > +
> > +/**
> > + * Gives an app access to all the currently configured data services.
> > + *
> > + * @author Burch
> > + *
> > + */
> > +public interface IAccessContext {	
>
> Your functions do exist, I merged before PR'ing, view whole file? I think
> this only shows the diffs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IKANOW/Aleph2/pull/1/files#r29862783>.
>